public/scss/partials/_bare.scss

```diff
@@ -52,7 +52,7 @@
 .bare__description {
   font-size: .8em;
   height: 100%;
-  padding: 20px;
+  padding: 2.5em;
```

Comment:
this should be 20px because if it is in ems, and the 3 columns have different font-sizes, the margin are going to be different on each column, but if we use pixels the margin will be the same